src/hbbft.erl

```diff
@@ -346,7 +376,20 @@ handle_msg(Data = #hbbft_data{round=R}, J, {dec, R, I, Share}) ->
                                     check_completion(Data#hbbft_data{dec_shares=NewShares, decrypted=NewDecrypted,
                                                                     failed_decrypt=[I|Data#hbbft_data.failed_decrypt]});
                                 Decrypted ->
+                                    #hbbft_data{batch_size=B, n=N} = Data,
```

Comment:
Just nitpicking and I know we don't do it anywhere in the module (we should have) but it would be nice to have accessor funs for `hbbft_data` record imo. So like `batch_size(Data)`, `num_nodes(Data)` etc perhaps? Similarly, we can then just have `B div N` calculated in a separate fun or whatever and check whether the transactions length exceeds.

Maybe a thing to do as a simplification pass in another PR

Reply:
I like it.

Definitely better done in a separate PR though, since it'll be a much larger diff and will further obscure the point of this one (I already did some nonessential things, like reformatting the signature (though it kind of was essential for my reading of it)).

Reply:
File an issue to remind us?

Reply:
@Vagabond Done: https://github.com/helium/erlang-hbbft/issues/64